amr-wind/equation_systems/icns/source_terms/ABLForcing.cpp

```diff
@@ -19,7 +20,26 @@ ABLForcing::ABLForcing(const CFDSim& sim) : m_time(sim.time())
     // TODO: Allow forcing at multiple heights
     pp_abl.get("abl_forcing_height", m_forcing_height);
     amrex::ParmParse pp_incflo("incflo");
-    pp_incflo.getarr("velocity", m_target_vel);
+
+    pp_abl.query("velocity_timetable", m_vel_timetable);
+    if (!m_vel_timetable.empty()) {
+        std::ifstream ifh(m_vel_timetable, std::ios::in);
```

Comment:
Make sure you include something to do the end of line. See here https://github.com/Exawind/amr-wind/blob/3d2e837f51a8c3a403c359d40264b21bf5c88850/amr-wind/utilities/tagging/CartBoxRefinement.cpp#L38